### ISSUE TITLE
Add mongodb-tools

### ISSUE THREAD

**Issue by yosmanyga** [mvertes/docker-alpine-mongo#17]
It would be nice to have mongodump and mongorestore.

**Comment by mlippens** [comment]
I don't think this is possible, there is no mongodb-tools package in alpine (AFAK)

**Comment by montera82** [comment]
in my case, my api image was based of golang, 
`git clone https://github.com/mongodb/mongo-tools`
and ran `go install`
was easy for me since i was gona use it from the api side

**Comment by mvertes** [comment]
Inside this image, you could just add mongo tools:
`docker exec -ti mongo sh`
then in the shell:
`apk add --no-cache mongodb-tools`

**Comment by mbana** [comment]
please add these https://docs.mongodb.com/manual/reference/program/ as mentioned by others.
